src/styles/components/_form.scss

```diff
@@ -58,7 +58,7 @@ form,
 form.dso-single-page .dso-form-buttons,
```

Comment:
deze hele trits is toch nu volledig dubbelop met https://github.com/dso-toolkit/dso-toolkit/pull/871/files#diff-4fc97984494ab789ee850ba76e8767bf9821ccd43c6d89497978b15e1a7fd6e7R25 ?